source/internal/_xuniqWith.js

```diff
@@ -0,0 +1,24 @@
+import _Set from './_Set.js';
+import _curry2 from './_curry2.js';
+import _includesWith from './_includesWith.js';
+import _xfBase from './_xfBase.js';
+
+
+function XUniqWith(pred, xf) {
+  this.xf = xf;
+  this.pred = pred;
+  this.items = [];
+}
+XUniqWith.prototype['@@transducer/init'] = _xfBase.init;
+XUniqWith.prototype['@@transducer/result'] = _xfBase.result;
+XUniqWith.prototype['@@transducer/step'] = function(result, input) {
+  if (_includesWith(this.pred, input, this.items)) {
+    return result;
+  } else {
+    this.items[this.items.length] = input;
```

Comment:
Was there a reason for not doing `this.items.push(input)`?

Reply:
No, I just saw somewhere in the source code did the same thing, and I picked it.
Array#push is definitely more readable and intuitive. I've updated this to use Array#push.